src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -186,6 +188,17 @@ const tickLabelClickDebounce = debounce(
     },
 );
 
+function sanitizeContextPoints(
+    chartType: ChartType,
+    points: IHighchartsPointObject[],
+): IHighchartsPointObject[] {
+    if (!isHeatmap(chartType)) {
```

Comment:
This is just a minor thing but I would rather remove this negation and return `points.filter..` and then `return points;` at the end of the method. This is how it is usually written. If you decide to add another condition for another chart type the method would have to be refactored in a way I suggest anyway.

Reply:
@xMort, I have updated the code, please help me review again
Thanks